backend/src/tests/test_crud.py

```diff
@@ -7,11 +7,11 @@
 from .fake_models import FakeTodoDAO, FakeUserDAO
 
 
-def make_todo(_id: int, title: str, content: str, user: UserDAO):
+async def make_todo(_id: int, title: str, content: str, user: UserDAO):
     "Helper function to simulate writing Todo's to database"
     todo = TodoDAO(id=uuid.UUID(int=_id), title=title, content=content, user=user)
     db_session = MagicMock()
-    FakeTodoDAO.create(db_session, todo)
+    await FakeTodoDAO.create(db_session, todo)
     return todo
 
 
```

Comment:
@pytest.mark.asyncio

Reply:
This function isn't actually a test, just used in other tests.  I think I have this refactored out of existence in another branch so it's going away..